### ISSUE TITLE
Feature: Add flag to import to use what's already stored in database.ts

### ISSUE THREAD

**Issue by TroyMorvant** [exogee-technology/graphweaver#1033]
**TLDR**
So what would be nice is if there were another flag added to the import CLI command to tell it to use the existing database.ts file to generate the entities and as such not prompt the user for this information.

**Additional context**
Rather than storing the actual database connection information in `database.ts`.  However, given that I already have my database environment variable defined using a URL it would be nice if GraphWeaver would support a connection string as a url (ie: `postgreSQL://postgres:postgres@localhost:5432/db`).  I tried using MikrOrm's `clientUrl` property but it complained that the RDS Postgres instance didn't support SSL connections? 

example
```typescript
export const connection = {
	connectionManagerId: 'postgresql',
	mikroOrmConfig: {
		entities: entities,
		driver: PostgreSqlDriver,
		clientUrl: process.env['DATABASE_URL'], // postgresql://postgres:postgres@localhost:5432/db
	},
};
```

Produced this error:
```
{"level":30,"time":1723663557095,"pid":40525,"hostname":"Troys-Cold-Climate-Macbook.local","name":"graphweaver","msg":"Starting standalone server on :::9001"}
/Users/troy/WebstormProjects/cold-graphql-api/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:77
          return self.emit('error', new Error('The server does not support SSL connections'))
                                    ^

Error: The server does not support SSL connections
    at Socket.<anonymous> (/Users/troy/WebstormProjects/cold-graphql-api/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:77:37)
    at Object.onceWrapper (node:events:629:26)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)

```

I included a script to parse the url myself in `database.ts` like so:

```typescript
import { PostgreSqlDriver } from '@mikro-orm/postgresql';
import { entities } from './entities';

const connectionValues = () => {
	const url = process.env.DATABASE_URL;
	if (url) {
		const match = url.match(/postgresql:\/\/([^:]+):([^@]+)@([^:]+):(\d+)\/(.+)/);
		if (match) {
			const [, user, password, host, port, dbName] = match;
			return { user, password, host, port, dbName };
		}
	}

	throw new Error('DATABASE_URL is not set');
}

export const connection = {
	connectionManagerId: 'postgresql',
	mikroOrmConfig: {
		driverOptions: {
			connection: { ssl: { rejectUnauthorized: false } },
		},
		entities: entities,
		driver: PostgreSqlDriver,
		dbName: connectionValues().dbName,
		host: connectionValues().host,
		user: connectionValues().user,
		password: connectionValues().password,
		port: +connectionValues().port,
	},
}
export const connections = [connection];
```

But since this is subject to being overwritten  (in case someone sets the overwrite flag on import) it would be nice if the code that is generated would already grab from the environment variables by default as we don't want a developer accidentally overwriting this file with actual db credentials and checking it into github.

```typescript
export const connection = {
	connectionManagerId: 'postgresql',
	mikroOrmConfig: {
		entities: entities,
		driver: PostgreSqlDriver,
		dbName: process.env['DATABASE_NAME'],
		host: process.env['DATABASE_HOST'],
		user: process.env['DATABASE_USER'],
		password: process.env['DATABASE_PASSWORD'],
		port: process.env['DATABASE_PORT'],
	},
};
```

In either case once the variables are stored in `database.ts` I shouldn't have to supply them to the import CLI in order to generate new entities.

**Comment by thekevinbrown** [comment]
This makes perfect sense, thanks for the comprehensive write up. I'll see if I can sneak it in shortly. If nothing else, reading the `.env` when we run the CLI and skipping the questions we already have answers to would make sense.
